File: Python/B2_7600.py
# 7600 문자가 몇갤까
while True:
    str = input().lower()
    if str == "#":
        break
    arr = [0] * 26
    for i in range(0, len(str)):
        ch = str[i]
        if ch.isalpha():
            # 문자를 아스키코드로 변환 == 자바의 charAt()
            index = ord(ch) - ord('a')
            arr[index] += 1

    cnt = 0
    for i in range(0,26):
        if arr[i] > 0:
            cnt += 1

    print(cnt)

File: B2_7600.java
import java.util.Scanner;
// 7600 문자가 몇갤까
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while (true) {
            String str = sc.nextLine();
            if(str.equals("#")) {
                break;
            }
            int[] arr = new int[26];
            str = str.toLowerCase();
            for(int i=0; i<str.length(); i++) {
                char ch = str.charAt(i);
                // ch가 문자인지 확인
                if(Character.isLetter(ch)) {
                    int index = ch - 'a';
                    arr[index]++;
                }
            }
            int cnt = 0;
            for(int i=0; i<26; i++) {
                if(arr[i] > 0) {
                    cnt++;
                }
            }
            System.out.println(cnt);
        }
    }
}